spec/function.dd

```diff
@@ -670,11 +670,15 @@ $(H2 $(LNAME2 inout-functions, Inout Functions))
         ---
         )
 
-    $(P The $(D_KEYWORD inout) token forms a wildcard that stands in for
-        any of mutable, `const`, `immutable`, `inout`, or `inout const`. When calling
-        the function, the `inout` of the return type is changed to whatever
-        the mutable, `const`, `immutable`, `inout`, or `inout const` status of the
-        argument type to the `inout` parameter was.
+    $(P The $(D_KEYWORD inout) keyword forms a wildcard that stands in for
+        one of these modifiers: mutable, `const`, `immutable`, `inout`, or `inout const`.
+        When calling the function, the `inout` modifier of the return type is changed to
+        match one of the above modifiers in the argument type to the `inout` parameter.
```

Comment:
I think this line is difficult to parse. One way to read it is that all of the above modifiers are "in" the argument type and one of them is being picked. Then the "to" could be read as "the modifier of the argument type is being matched to the parameter". I know the intended point, but I still stumbled over it on the first pass. And I know the "to" is from the original sentence. It was awkward then, but more obvious now with your new wording.

I think if you replace "match one of the above modifiers in" to "that of" and insert "passed" before "to", it's more readable:

"the inout state of the return type is changed to that of the argument type passed to the inout parameter".